usr/share/rear/output/PXE/default/801_copy_to_http.sh

```diff
@@ -0,0 +1,52 @@
+# 801_copy_to_http.sh
+#
+# copy kernel and initrd to HTTP server for Relax-and-Recover
+#
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+# Nothing to do here when PXE_HTTP_UPLOAD_URL is not set
+if [[ -z "$PXE_HTTP_UPLOAD_URL" ]] ; then
+    DebugPrint "PXE_HTTP_UPLOAD_URL not set, skipping copy"
```

Comment:
Please remove this after you are happy, we don't have those kind of debug infos in other scripts (or we shouldn't)

Many scripts start with `[[ "foobar" ]] || return` to run only for specific use cases